test/dotnet.Tests/GivenThatTheUserIsRunningDotNetForTheFirstTime.cs

```diff
@@ -61,21 +61,10 @@ public void UsingDotnetForTheFirstTimeWithNonVerbsDoesNotPrintEula()
         [Fact]
         public void ItShowsTheAppropriateMessageToTheUser()
         {
-            string firstTimeUseWelcomeMessage = NormalizeLineEndings(@"Welcome to .NET Core!
----------------------
-Learn more about .NET Core @ https://aka.ms/dotnet-docs. Use dotnet --help to see available commands or go to https://aka.ms/dotnet-cli-docs.
-
-Telemetry
---------------
-The .NET Core tools collect usage data in order to improve your experience. The data is anonymous and does not include command-line arguments. The data is collected by Microsoft and shared with the community.
-You can opt out of telemetry by setting a DOTNET_CLI_TELEMETRY_OPTOUT environment variable to 1 using your favorite shell.
-You can read more about .NET Core tools telemetry @ https://aka.ms/dotnet-cli-telemetry.
-
-Configuring...
--------------------
-A command is running to initially populate your local package cache, to improve restore speed and enable offline access. This command will take up to a minute to complete and will only happen once.");
-
-            // normalizing line endings as git is occasionally replacing line endings in this file causing this test to fail
+            string firstTimeUseWelcomeMessage = NormalizeLineEndings(Configurer.LocalizableStrings.FirstTimeWelcomeMessage);
+            // normalizing line endings as what is used in the resources may
+            // does not necessarily match how the command stdout capturing logic 
+            // handles newlines.
             NormalizeLineEndings(_firstDotnetVerbUseCommandResult.StdOut)
                 .Should().Contain(firstTimeUseWelcomeMessage)
                      .And.NotContain("Restore completed in");
```

Comment:
Yes. We'll have to do some fixup when nuget satellites are ingested. Mind you, NotContain checks like this won't be caught by test failures. I think it is ok if we only have negative message coverage like this in en case since there isn't too much of this.